website/releases/release-0.11.0.md

```diff
@@ -0,0 +1,206 @@
+
+- Deployment Model A: If your current deployment model is single writer and all table services (cleaning, clustering,
+  compaction) are configured to be **inline**, such as Deltastreamer sync-once mode and Spark Datasource with default
+  configs, there is no additional configuration required. Restarting the Single Writer with 0.11.0 is sufficient to
+  safely enable metadata table.
+- Deployment Model B: If your current deployment model is single writer along with async table services (such as
+  cleaning, clustering, compaction) running in the same process, such as `HoodieDeltaStreamer` continuous mode writing
+  MOR table, Spark streaming (where compaction is async by default), and your own job setup enabling async table
+  services inside the same writer, it is a must to have the optimistic concurrency control, the lock provider, lazy
+  failed write clean policy configured before enabling metadata table. More details can be found [here](/docs/metadata).
+  Failing to follow the configuration guide leads to loss of data.
+- Deployment Model C: If your current deployment model is [multi-writer](/docs/concurrency_control) along
+  with [a lock provider](/docs/concurrency_control/#enabling-multi-writing) and other required configs set for every
+  writer, there is no additional configuration required. You can bring up the writers with 0.11.0 sequentially. Applying
+  the proper configurations to only partial writers leads to loss of data from the inconsistent writer. So, ensure you
+  enable metadata table across all writers.
+
+### Use async indexer
+
+Configuring a lock provider is a prerequisite for using async indexer. The implementation details were illustrated
+in [RFC-45](https://github.com/apache/hudi/blob/master/rfc/rfc-45/rfc-45.md). At the minimum, users need to set the
+following configurations to schedule and run the indexer:
+
+```shell
+# enable async index
+hoodie.metadata.index.async=true
+# enable specific index type, column stats for example 
+hoodie.metadata.index.column.stats.enable=true
+# set OCC concurrency mode
+hoodie.write.concurrency.mode=optimistic_concurrency_control
+# set lock provider configs
+hoodie.write.lock.provider=<LockProviderClass>
+```
+
+Few points to note from deployment perspective:
+
+1. Files index is created by default as long as the metadata table is enabled.
+2. If you intend to build any index asynchronously, say column stats, then be sure to enable the async index and column
+   stats index type on the regular ingestion writers as well.
+3. In the case of multi-writers, enable async index and specific index config for all writers.
+4. While an index can be created concurrently with ingestion, it cannot be dropped concurrently. Please stop all writers
+   before dropping an index.
+
+Some of these limitations will be overcome in the upcoming releases. Please
+follow [HUDI-2488](https://issues.apache.org/jira/browse/HUDI-2488) for developments on this feature.
+
```

Comment:
should we add the spark-avro package requirement relaxing in migration section ? I see its covered in highlights.

Reply:
I see we are missing some callouts we discussed. 
1. setting right preCombine field for MOR is mandatory. if not, queries could fail
2. read config about disabling appending partition values to dataframe read on spark https://github.com/apache/hudi/pull/5377

Reply:
Added the highlights in the migration guide